scripts/generate_version_docs.py

```diff
@@ -109,7 +109,7 @@ def get_static_docs():
     },
     {
       "id": 'terra-jupyter-gatk_legacy',
-      "label": 'Legacy GATK (GATK 4.1.4.1, Python 3.7.7, R 3.6.3)',
+      "label": 'Legacy GATK (default prior to May XX, 2020) (GATK 4.1.4.1, Python 3.7.7, R 3.6.3)',
```

Comment:
I wonder if we should use a shorter date format since the label is already a bit long